com.unity.multiplayer.mlapi/Runtime/Spawning/NetworkSpawnManager.cs

```diff
@@ -25,9 +25,6 @@ public class NetworkSpawnManager
         /// </summary>
         public readonly Dictionary<ulong, NetworkObject> SpawnedObjects = new Dictionary<ulong, NetworkObject>();
```

Comment:
I guess we will eventually get rid of `NetworkSpawnManager` entirely at some point. For instance, `SpawnedObjects` should probably be managed by `NetworkSceneManager` since they are properties of scenes. Even if we were to move objects between scenes, they are still "in" scenes. Managing object spawning and instances in 2 (`NetworkSpawnManager` & `NetworkSceneManager`) feels a bit weird to me :P

But obv, that's something for later — just wanted to braindump my 2-cents here.

Reply:
Yeah, it is weird indeed.  This update removes the dependency from NetworkSpawnManager and places it into NetworkSceneManager in preparation for the proposed direction.  If we do go in the proposed general direction, then the object synchronization process will be handled differently anyway.... so keeping a list of in-scene NetworkObjects instantiated via scene loading really isn't "handling spawning"... it is just "tracking what was automatically instantiated".